support/package_internal_ext

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+if [ -z "$1" ]; then
+    echo "Usage: package_ext EXT API..."
+    exit 1
+fi
+
+basedir="$( cd -P "$( dirname "$0" )" && pwd )"
+source "$basedir/../conf/buildpack.conf"
+source "$basedir/lib/utils"
+source "$basedir/lib/swift"
+export PATH=${basedir}/../vendor/bin:$PATH
+
+indent() {
```

Comment:
Why do you copy the `indent` and `build_ext` function in this script instead of `source`-ing the file it is already defined in?

Reply:
build_ext is not the same, I'm going to make it clearer